File: scratch_test.mjs
import { createClient } from '@supabase/supabase-js';
import dotenv from 'dotenv';
dotenv.config({ path: 'client/.env' });

const supabase = createClient(process.env.VITE_SUPABASE_URL, process.env.VITE_SUPABASE_ANON_KEY);

async function main() {
  const { data, error } = await supabase.from('matches').select('*').limit(1);
  console.log("Error:", error);
  console.log("Data:", data);
}
main();

File: scratch_test.py
import asyncio
from supabase import create_client, Client
import os
from dotenv import load_dotenv

load_dotenv('client/.env')

url = os.environ.get("VITE_SUPABASE_URL")
key = os.environ.get("VITE_SUPABASE_ANON_KEY")

supabase: Client = create_client(url, key)

response = supabase.table("matches").select("*").limit(1).execute()
print(response)
